src/meta/api/endpoint/ApiEndPoint.ts

```diff
@@ -15,6 +15,10 @@ export const ApiEndPoint = {
     areas: () => apiPath('area', 'areas'),
   },
 
+  Assessment: {
```

Comment:
better to move this under metadata/assessment. 
also, can you verify if Sections is still needed ?